weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -872,6 +967,49 @@ def table_query_stats(self, req: tsi.TableQueryStatsReq) -> tsi.TableQueryStatsR
 
         return tsi.TableQueryStatsRes(count=count)
 
+    def _table_delete(self, project_id: str, digest: str) -> None:
```

Comment:
This is going to need to be smarter in that it only deletes rows once they no longer are used by any other table

Reply:
Ugh right....